### ISSUE TITLE
Publish to npm

### ISSUE THREAD

**Issue by blake-regalia** [cleishm/express-negotiate#4]
The latest tarball on npm shows version 0.0.5 in package.json however the main export reports `'0.0.4'` and is using the deprecated `mime.lookup`, clearly out of sync with the GH repo.

**Comment by cleishm** [comment]
Yes, I believe someone independently pushed an update. Indeed, it's at `0.0.6` on npmjs now.

It would be great to update this repo to the latest and push from here. Happy to give access to anyone working on that.
